src/sources/file.rs

```diff
@@ -396,7 +396,7 @@ pub fn file_source(
             }
             event
         });
-        tokio::spawn(async move { out.send_stream(&mut messages).instrument(span).await });
+        tokio::spawn(async move { out.send_event_stream(&mut messages).instrument(span).await });
```

Comment:
Can we lose the `async move { ... }.await` here?

Reply:
I don't think so. `tokio::spawn` requires all data be `'static`, which is why we need the `move`.